code/game/mecha/combat/durand.dm

```diff
@@ -9,7 +9,7 @@
 	icon_state = "durand"
 	step_in = 4
 	dir_in = 1 //Facing North.
-	max_integrity = 600
```

Comment:
I think that's a bit too drastic..? Make it 500 at least, otherwise we'll drop down entirely to pre-buff state, where mechs felt like being made of paper.